pkg/core/interop/contract/call.go

```diff
@@ -74,6 +96,7 @@ func callExFromNative(ic *interop.Context, caller util.Uint160, cs *state.Contra
 
 	ic.VM.Invocations[cs.Hash]++
 	ic.VM.LoadScriptWithCallingHash(caller, cs.NEF.Script, cs.Hash, ic.VM.Context().GetCallFlags()&f, true, uint16(len(args)))
+	ic.VM.Context().NEF = &cs.NEF
```

Comment:
Can we pass it with `LoadScriptWithCallingHash`? And take Hash and Script from it.

Reply:
We can't pass contract state (circular dependencies) and NEF contains no hash. It is also used a lot in tests with arbitrary scripts.